src/runtime/cmd/kata-monitor/README.md

```diff
@@ -0,0 +1,59 @@
+  * Kata agent metrics
+  * Kata guest OS metrics
+  * Hypervisor metrics
+  * Firecracker metrics
+  * Kata monitor metrics
+
+All the provided metrics are in Prometheus format. While Kata monitor can be used as a standalone daemon on any host running Kata Containers workloads and can be used for retrieving profiling data from the running Kata runtimes, its main expected usage is to be deployed as a Daemonset on a Kubernetes cluster: there Prometheus should scrape the metrics from the kata-monitor endpoints.
+For more information on the Kata Containers metrics architecture and a detailed list of the available metrics provided by Kata monitor check the [Kata 2.0 Metrics Design](../../../../docs/design/kata-2-0-metrics.md) document.
+
+## Usage
+Each Kata monitor instance detects and monitors the Kata Container workloads running on the same node.
+
+### Kata monitor args
+The `kata-monitor` binary accepts the following arguments:
+
+* `--listen-address` _IP:PORT_
+* `--runtime-enpoint` _PATH_TO_THE_CONTAINER_MANAGER_CRI_INTERFACE_
+* `--log-level` {trace|debug|info|warn|error|fatal|panic}
+
+The **listen-address** specifies the IP and tcp port where the kata-monitor HTTP endpoints will be exposed. It defaults to `127.0.0.1:8090`.
+
+The **runtime-endpoint** is the CRI of a CRI compliant container manager: it will be used to retrieve the CRI PodSandboxMetadata (uid, name, namespace) which will be attached to the Kata metrics through the labels _cri_uid_, _cri_name_ and _cri_namespace_. It defaults to the containerd socket: `/run/containerd/containerd.sock`.
+
+The **log-level** allows the chose how verbose the logs should be. The default is `info`.
+### Kata monitor HTTP endpoints
+`kata-monitor` exposes the following endpoints:
+  * `/metrics`             : get Kata sandboxes metrics.
+  * `/sandboxes`           : list all the Kata sandboxes running on the host.
+  * `/agent-url`           : Get the agent URL of a Kata sandbox.
+  * `/debug/vars`          : Golang pprof _/debug/vars_ endpoint for the Kata runtime shim process.
+  * `/debug/pprof/`        : Golang pprof _/debug/pprof/_ endpoint for the Kata runtime shim process.
+  * `/debug/pprof/cmdline` : Golang pprof _/debug/pprof/cmdline_ endpoint for the Kata runtime shim process.
+  * `/debug/pprof/profile` : Golang pprof _/debug/pprof/profile_ endpoint for the Kata runtime shim process.
+  * `/debug/pprof/symbol`  : Golang pprof _/debug/pprof/symbol_ endpoint for the Kata runtime shim process.
+  * `/debug/pprof/trace`   : Golang pprof _/debug/pprof/trace_ endpoint for the Kata runtime shim process.
+
+**NOTE: The debug endpoints are available only if the [Kata Containers configuration file](../../config/configuration-qemu.toml.in#L590-L592) includes** `enable_pprof = true` **in the** `[runtime]` **section**.
+
+The `/sandboxes` endpoint lists the sandbox id of all the detected Kata runtimes. If accessed via a web browser, it provides html links to the endpoints available for each sandbox.
```

Comment:
s/id/ID unless it is a variable name, then make some indication such as you do below

Reply:
fixed, thanks!